webapp/templates/admin_upload.html

```diff
@@ -42,6 +42,7 @@
 <div class="column">
   <h2>File payload</h2>
   <form method="POST" action="/api/results/upload" enctype="multipart/form-data">
+    {{if .CallbackURL}}<input style="display:none;" name="callback_url" value="{{.CallbackURL}}">{{end}}
```

Comment:
Why not `type="hidden"` ?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden